api/nodes/utils.py

```diff
@@ -25,6 +25,9 @@ def get_file_object(node, path, provider, request):
             )
         return obj
 
+    if not node.has_addon(provider):
```

Comment:
This doesn't work if the addon is authorized but not configured.

<img width="285" alt="screen shot 2016-05-02 at 09 55 41" src="https://cloud.githubusercontent.com/assets/5659262/14956116/1c2d384e-104c-11e6-9d1e-1300d37ec713.png">
<img width="571" alt="screen shot 2016-05-02 at 09 55 54" src="https://cloud.githubusercontent.com/assets/5659262/14956115/1c2c9df8-104c-11e6-83a1-297cf52c3d83.png">


You can check that with `node.get_addon(provider).configured`